mappings/net/minecraft/client/gui/screen/GameModeSelectionScreen.mapping

```diff
@@ -0,0 +1,46 @@
+CLASS net/minecraft/class_5289 net/minecraft/client/gui/screen/GameModeSelectionScreen
+	FIELD field_24566 TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_24567 UI_WIDTH I
+	FIELD field_24568 currentGameMode Ljava/util/Optional;
+	FIELD field_24569 gameMode Ljava/util/Optional;
+	FIELD field_24570 lastMouseX I
+	FIELD field_24571 lastMouseY I
+	FIELD field_24572 mouseUsedForSelection Z
+	FIELD field_24573 gameModeButtons Ljava/util/List;
+	METHOD method_28063 drawHotkeys (Lnet/minecraft/class_4587;Ljava/lang/String;Ljava/lang/String;II)V
+		ARG 2 keyName
+		ARG 3 action
+		ARG 4 x
+		ARG 5 y
+	METHOD method_28064 apply (Lnet/minecraft/class_310;Ljava/util/Optional;)V
+		ARG 0 client
+		ARG 1 gameMode
+	METHOD method_28068 apply ()V
+	METHOD method_28069 checkForClose ()Z
+	CLASS class_5290 GameMode
```

Comment:
how does this one differ from the existing gamemode enum? may this be confusing?

Reply:
It's not going to be confusing, because this is an internal inner class used only in the selection screen.

Reply:
what else can you name this? `SelectionEntry` lol?

Reply:
As long as there is no import conflicts I am fine (maybe you need to check decompiled sources)